spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/ProtobufUtilsTest.java

```diff
@@ -115,43 +115,51 @@ public void testBigQueryToProtoSchema() throws Exception {
   public void testSparkStructRowToDynamicMessage() throws Exception {
     StructType schema = new StructType().add(SPARK_NESTED_STRUCT_FIELD);
     Descriptors.Descriptor schemaDescriptor = toDescriptor(schema);
-    Message converted = buildSingleRowMessage(schema, schemaDescriptor, STRUCT_INTERNAL_ROW);
+    Message converted =
+        buildSingleRowMessage(
+            schema,
+            schemaDescriptor,
+            STRUCT_INTERNAL_ROW,
+            /*fieldIndexToEntryMap*/ null,
```

Comment:
Can we add a test to verify cached entries?

Reply:
I added an additional test here. Previously I was relying on the call to toProtoRows() with multiple input rows to test it.